components/atom/checkbox/src/index.scss

```diff
@@ -11,6 +11,7 @@ $m-atom-checkbox-type-right: $m-m !default;
 $bgc-atom-checkbox: $c-white !default;
 $bgc-checked-atom-checkbox: $c-primary !default;
 $bd-atom-checkbox: 1px solid $c-primary !default;
+$bd-atom-checkbox--checked: 1px solid $c-primary !default;
```

Comment:
Instead of 1px, use https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss#L12 👍

Reply:
done! ✅